src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -33,8 +33,8 @@ import scala.concurrent.Future
 object Parser extends LazyLogging {
 
   type ParseSuccessResult = (Vector[Source], PPackage)
-  type ParseResult = Either[Vector[ParserError], ParseSuccessResult]
-  type ImportToSourceOrErrorMap = Vector[(AbstractPackage, Either[Vector[ParserError], Vector[Source]])]
+  type ParseResult = Either[Vector[VerifierError], ParseSuccessResult]
```

Comment:
I find the name a bit confusing, in that I didn't expect the parser to return `VerifierError` (as this makes it sound like sth failed during verification). Is there a reason why having `ParseError` here is cumbersome or not possible? Otherwise, is `ParseError` still used anywhere else? Because, if it isn't, then it should be deprecated

Reply:
(I will continue the review after this comment is answered)